### PR TITLE
integrate the current state of the new antelope library with no effects

### LINKED ISSUE #202: integrate the current state of the new Antelope library with no effects
## Overview
- As a user, I want the site to load at the same speed or faster than before the Antelope library was integrated.
- As a developer, I want to ensure that the Antelope library is fully integrated and functioning as expected without causing any errors or bugs in the site's existing codebase.
- As a developer, I want to ensure that the Antelope library is implemented in a way that does not interfere with any other libraries or tools that are already being used on the site.

## Acceptance Criteria
- The website must load at the same speed or faster than before the Antelope library was integrated.
- All existing features on the website must continue to function properly without any negative effects or changes in performance.
- The Antelope library must be fully integrated and functioning as expected without causing any errors or bugs in the site's existing codebase.
- The Antelope library must be implemented in a way that does not interfere with any other libraries or tools that are already being used on the site.

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #202

## Description
This work is the continuation of [this PR](https://github.com/telosnetwork/telos-wallet/pull/201) which incorporates the Antelope library we are creating based on the Pinia store implementation and changes some of the current components to show new functionalities from the library. However, this PR removes all changes made to those components and **only** incorporates a functioning but still not a complete library, ready to keep growing to incorporate more functionalities to support EVM requirements.

## Test scenarios
Nothing should change for a user

## Checklist:
-   [x] I have performed a self-review of my own code
-   [x] I have commented on my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all English text to the translation file
-   [x] I have created a new issue with the required translations for the currently supported languages
-   [x] I have added appropriate test coverage

### CODE REVIEW

File: src/antelope/chains/EVMChain.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { PriceChartData, Token } from 'src/types';
+// import { Theme } from 'src/types';
```
Comment (EJSG) [inline src/antelope/chains/EVMChain.ts:2]:
```suggestion

```

File: src/antelope/stores/account.ts [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+                    localStorage.setItem('network', network);
+                    localStorage.setItem('account', account);
+                    localStorage.setItem('isNative', 'false');
+                    localStorage.setItem('autoLogin', 'evm');
+                    success = true;
+                    this.fetchAccountDataFor('logged', evmAccount);
+                    getAntelope().events.onLoggedIn.next(evmAccount);
+                } else {
+                    // TODO: check when and how does this error happends if ever
```
Comment (EJSG) [inline src/antelope/stores/account.ts:161]: TODO should be converted to issue or removed, so we don't lose it

File: src/antelope/stores/account.ts [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+                if (this.__logged_account?.isNative) {
+                    const logged = this.__logged_account as NativeAccountModel;
+                    const { authenticator } = logged;
+                    try {
+                        authenticator && (await authenticator.logout());
+                    } catch (error) {
+                        console.error('Authenticator logout error', error);
+                    }
+                } else {
+                    // useEVMStore().logout(); ?
```
Comment (EJSG) [inline src/antelope/stores/account.ts:189]: maybe make an issue for this so we don't lose it?

File: src/antelope/stores/account.ts [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+                console.error('Error: ', errorToString(error));
+            } finally {
+                useFeedbackStore().unsetLoading('account.autoLogin');
+            }
+        },
+        async sendAction({ account, data, name, actor, permission }: SendActionData) {
+            this.trace('sendAction', account, data, name, actor, permission);
+            try {
+                useFeedbackStore().setLoading('account.sendAction');
+                // TODO: creates a list of one transaction and use the sendTransaction() function
```
Comment (EJSG) [inline src/antelope/stores/account.ts:238]: issue or remove

File: src/antelope/stores/account.ts [hunk 5]
```diff
@@ -0,0 +1,334 @@
...
+                console.error('Error: ', errorToString(error));
+            } finally {
+                useFeedbackStore().unsetLoading('account.sendAction');
+            }
+        },
+        async sendTransaction(actions: Action[]) {
+            this.trace('sendTransaction', actions);
+            try {
+                useFeedbackStore().setLoading('account.sendTransaction');
+                // TODO: creates a transaction with the actions and interacts with wallet and try to send a transaction
```
Comment (EJSG) [inline src/antelope/stores/account.ts:250]: issue or remove

File: src/antelope/stores/chain.ts [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+// test native chains
+import TelosTestnet from 'src/antelope/chains//native/telos-testnet';
+import Jungle from 'src/antelope/chains//native/jungle';
+
+// main evm chains
+import TelosEVM from 'src/antelope/chains//evm/telos-evm';
+
+// test evm chains
+import TelosTestnetEVM from 'src/antelope/chains//evm/telos-testnet-evm';
```
Comment (EJSG) [inline src/antelope/stores/chain.ts:40]: should these imports have the double slash? e.g. `'src/antelope/chains//native/ux'` between 'chains' and 'native'

File: src/antelope/stores/profile.ts [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+            try {
+                // TOOD: get profile data
+            } catch (error) {
+                console.error('Error: ', errorToString(error));
+            }
+        },
+        handleAccountLoggedOut() {
+            this.trace('handleAccountLoggedOut');
+            try {
+                // TOOD: clear profile data
```
Comment (EJSG) [inline src/antelope/stores/profile.ts:41]: issue or remove

File: src/antelope/stores/resources.ts [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+            try {
+                // TOOD: get resources data for the logged account
+            } catch (error) {
+                console.error('Error: ', errorToString(error));
+            }
+        },
+        handleAccountLoggedOut() {
+            this.trace('handleAccountLoggedOut');
+            try {
+                // TOOD: clear resources data
```
Comment (EJSG) [inline src/antelope/stores/resources.ts:47]: issue or remove

File: src/antelope/stores/rex.ts [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    getters: {
+    },
+    actions: {
+        trace: createTraceFunction(store_name),
+        init: createInitFunction(store_name),
+        // Logged Handlers ----------------
+        handleAccountLoggedIn(logged: AccountModel) {
+            this.trace('handleAccountLoggedIn', logged);
+            try {
+                // TOOD: get rex data for the logged account
```
Comment (EJSG) [inline src/antelope/stores/rex.ts:33]: issue or remove

**Comment by EJSG** [comment]
maybe for the  TODOs you can make a single issue for each store or each file which has remaining TODOs, instead of making a ton of smaller tickets.

**Review by EJSG — APPROVED** [review]
:fire: :fire:

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**